config.json

```
@@ -25,8 +25,8 @@
       "topics": [
         "control_flow_if_else_statements",
         "control_flow_loops",
-        "math",
-        "logic"
+        "logic",
+        "math"
```

```suggestion
        "math",
        "logic"
```
Not sure why these changes to unrelated exercises? If you'd like to tidy this up that's great but please do on a separate PR. This separation of PRs makes it easier to manage the repo's history